kramer/src/main/kotlin/kramer/Templates.kt

```diff
@@ -0,0 +1,50 @@
+package kramer
+
+internal fun aar_template(
```

Comment:
What's with the shift to snakecase? Had to look twice to ensure I wasn't in a bzl.

Reply:
Oh - sorry, it was originally a cut-paste from the .bzl. I'll fix.

Reply:
Done.